lib/models/calculator.js

```diff
@@ -173,6 +188,11 @@ module.exports = class Calculator {
           if (!resultsByPatient[patientId]) {
             resultsByPatient[patientId] = {};
           }
+
+          if (requestDocument) {
```

Comment:
If just sending back a 'document' formatted object, then perhaps just do nothing instead of mongoosing to just toObject it as it leaves the service.